.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
```

Comment:
Can we set this to 88 to comply with Black default style https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
Actually, let's just switch to Ruff.